src/core/RandomVariables.jl

```diff
@@ -1252,6 +1255,29 @@ function setorder!(mvi::TypedVarInfo, vn::VarName{sym}, index::Int) where {sym}
     return mvi
 end
 
+"""
+`getorder(vi::VarInfo, vn::VarName)`
+
+Some proper checks are missing!!
+"""
+
```

Comment:
The documentation should go right above the function definition (without additional lines in between). Also the standard in Julia is to indent the first line in the documentation block with 4 spaces.